ci-scripts/osx/travis-build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+pushd toonz/sources
+chmod +x beautification.sh
```

Comment:
I think you misunderstood. There's no need to do `chmod +x` after the checkout, you can simply mark it executable in the repository, as it is a script anyway.